assets/src/dashboard/parts/components/Modal.js

```diff
@@ -53,10 +53,16 @@ export default function Modal({	icon, labels = {}, onRequestClose = () => {}, on
 					className="text-center mx-0 my-4 text-gray-700"
 					dangerouslySetInnerHTML={ { __html: labels.description } }
 				/>
-
-				<Button variant="primary" className={ actionButtonClasses } onClick={ onConfirm }>
-					{ labels.action }
-				</Button>
+				<div class="flex gap-4">
+					<Button variant="primary" className={ actionButtonClasses } onClick={ onConfirm }>
+						{ labels.action }
+					</Button>
+					{ labels.action2 && (
+						<Button variant="link" className={ actionButtonClasses } onClick={ onAction2 }>
+							{ labels.action2 }
```

Comment:
Use `secondaryAction` for the label key.

Reply:
Done!